.github/workflows/back-end.yml

```diff
@@ -0,0 +1,228 @@
+            run: "composer exec --no-interaction -- parallel-lint src/ tests/"
+
+    unit_tests:
+        name: "2️⃣ Unit and functional tests"
+        needs:
+        - "byte_level"
+        - "syntax_errors"
+        strategy:
+            #fail-fast: false
+            matrix:
+                php-version:
+                - "8.1"
+                - "8.2"
+                dependencies:
+                - "lowest"
+                - "highest"
+        runs-on: "ubuntu-22.04"
+        timeout-minutes: 5
+        steps:
+        -
+            name: "Set up PHP"
+            uses: "shivammathur/setup-php@v2"
+            with:
+                php-version: "${{ matrix.php-version }}"
+        -
+            name: "Setup problem matcher for PHPUnit"
+            run: |
+                echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+        -
+            name: "Checkout repository"
+            uses: "actions/checkout@v3"
+        -
+            name: "Install dependencies"
+            uses: "ramsey/composer-install@v2"
+            with:
+                dependency-versions: "${{ matrix.dependencies }}"
+        -
+            name: "Execute unit tests"
+            run: "composer run unit"
+        -
+            name: "Execute feature tests"
+            run: "composer run feature"
+        -
+            name: "Send coverage to Coveralls"
+            env:
+                COVERALLS_REPO_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+            if: "${{ env.COVERALLS_REPO_TOKEN && matrix.php-version == '8.2' && matrix.dependencies == 'highest' }}"
+            run: |
+                wget --no-verbose \
+                    "https://github.com/php-coveralls/php-coveralls/releases/download/v2.5.3/php-coveralls.phar"
+                #php ./php-coveralls.phar --verbose
+
+    static_analysis:
+        name: "3️⃣ Static Analysis"
+        needs:
+        - "byte_level"
+        - "syntax_errors"
+        runs-on: "ubuntu-22.04"
+        timeout-minutes: 5
+        steps:
+        -
+            name: "Set up PHP"
+            uses: "shivammathur/setup-php@v2"
+            with:
+                php-version: "8.2"
+                coverage: "none"
+        -
+            name: "Checkout repository"
+            uses: "actions/checkout@v3"
+        -
+            name: "Check JSON files"
+            run: |
+                git ls-files --cached -z -- '*\.json' \
+                    | xargs -t --null -L 1 -- \
+                        php -r 'json_decode(file_get_contents($argv[1]), null, 512, JSON_THROW_ON_ERROR);'
+        -
+            name: "Validate Composer configuration"
+            run: "composer validate --no-interaction --strict"
+        -
+            name: "Install dependencies"
+            uses: "ramsey/composer-install@v2"
+            with:
+                dependency-versions: "highest"
+        -
+            name: "Check PSR-4 mapping 🐌"
+            run: "composer dump-autoload --optimize --strict-psr"
```

Comment:
When FQCN does not match the filesystem path 💣

Reply:
didn't know this option existed, great!